src/relentless/simulate/lammps.py

```diff
@@ -268,6 +303,17 @@ def pair_map(sim, pair):
         ]
         cmds += ["pair_style table linear {N}".format(N=Nr)]
 
+        # set exclusions if snap has topology
+        if snap.has_bonds() or snap.has_angles() or snap.has_dihedrals():
+            excl_12, excl_13, excl_14 = 1.0, 1.0, 1.0
+            if sim.potentials.pair.exclusions is not None:
+                if "bond" in sim.potentials.pair.exclusions:
```

Comment:
I think we said we were using 1-2 as the exclusion name, no?
```suggestion
                if "1-2" in sim.potentials.pair.exclusions:
```